apps/antalmanac/src/components/RightPane/SectionTable/EnrollmentHistoryPopup.tsx

```diff
@@ -64,32 +78,55 @@ export function EnrollmentHistoryPopup({ department, courseNumber }: EnrollmentH
         );
     }
 
+    const handleBack = () => {
+        setGraphIndex((prev) => prev - 1);
+    };
+
+    const handleForward = () => {
+        setGraphIndex((prev) => prev + 1);
+    };
+
     return (
         <Box sx={{ padding: 0.5 }}>
-            <Typography
+            <Box
                 sx={{
-                    marginTop: '.5rem',
-                    textAlign: 'center',
-                    fontWeight: 500,
-                    marginRight: '2rem',
-                    marginLeft: '2rem',
-                    marginBottom: '.5rem',
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                    width: graphWidth,
                 }}
             >
-                {popupTitle}
-            </Typography>
+                <Tooltip title="Newer Graph">
+                    {/* In order for a tooltip to work properly with disabled buttons, we need to wrap the button in a span */}
+                    <span>
+                        <IconButton onClick={handleBack} disabled={graphIndex === 0}>
+                            <ArrowBack />
+                        </IconButton>
+                    </span>
+                </Tooltip>
+                <Typography sx={{ fontWeight: 500, fontSize: isMobileScreen ? '0.8rem' : '1rem', textAlign: 'center' }}>
+                    {popupTitle}
+                </Typography>
+                <Tooltip title="Older Graph">
+                    <span>
+                        <IconButton onClick={handleForward} disabled={graphIndex === enrollmentHistory.length - 1}>
+                            <ArrowForward />
+                        </IconButton>
+                    </span>
+                </Tooltip>
+            </Box>
             <Link
                 href={`https://zot-tracker.herokuapp.com/?dept=${encodedDept}&number=${courseNumber}&courseType=all`}
```

Comment:
You could optionally define this as a variable prior to the return statement (optionally also memoized), so that the readability of the JSX is simpler.